list.js

```diff
@@ -39,9 +39,9 @@
 
         FILTERED_EVENTS = filterEvents(ALL_EVENTS, q);
         if (FILTERED_EVENTS.length === 0) {
-            $el.append('<div class="q-no-item">Unfortunately, there are ' +
-                'no results matching what you\'re looking for in ' +
-                'the Columbia Film Glossary content.</div>');
+            $el.append(`<div class="q-no-item">Unfortunately, there are
+                no results matching what you're looking for in
+                the CTL Events Calendar.</div>`);
```

Comment:
This new backtick syntax is cool but let's not use it unless we need to, because it has only been added recently to the javascript spec. For example, it doesn't work in Safari 8, which I know Zarina uses:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility

I know we're already using it in ctlevent.js, but let's keep it contained to that area of the code, and we can easily convert that to Mustache when necessary.